Cargo.toml

```diff
@@ -235,7 +235,7 @@ datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "b6
 datafusion-functions = { git = "https://github.com/spiceai/datafusion.git", rev = "b687ce48fdbdb77c055b0a011d746ff3f1167471" }       # spiceai/spiceai-49
 
 datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "5ad2f52b9bafc6eaa50851f2e1fcf0585fb5184d" }                      # spiceai-49
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "4cf111b947dc42efa945aa4d0250b02ee615273e" } # spiceai
+datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "3f4b4ac831068e7638ac4dbee8115e5c8271edea" } # phillip/251002-fix-blocking-health-check
```

Comment:
Update once https://github.com/datafusion-contrib/datafusion-table-providers/pull/450 is merged